### DISCUSSION TITLE
Analyzing SERP on Large Scale

### DISCUSSION THREAD

**Discussion by Rushikesh494** [eliasdabbas/advertools#128] [Ideas]
Hi, @eliasdabbas thank you very much, It was a really amazing tool. Is there any way to get top 50 results in 1 search request for a particular keyword? it was really helpful to all the digital marketers if we can pull the top 50 results in 1 search request. Since most SEO marketers would look at the bulk keyword tool, 10 requests for a keyword were taking too many requests. could you please work on it, please?

**Comment by eliasdabbas** [comment]
Thanks @Rushikesh494. Glad you liked it. 

Yes, this tool's main feature is that it retrieves bulk SERPs with one function call, including combinations. 
Unfortunately, Google only allows a maximum of ten results per request, so the 50 have to be split into five requests.
